core/kernel/dt.c

```diff
@@ -159,6 +159,57 @@ int dt_map_dev(const void *fdt, int offs, vaddr_t *base, size_t *size)
 	return 0;
 }
 
+#ifdef CFG_TPM_API
+/*
+ * This function retrieves the TPM Event log information.
+ * For security, the secure address of the event log will be zeroed after reading it to minimize
+ * any leak of information to the ns world, so care must be taken not to call this
+ * function more than once.
+ */
+
+#define PROP_TPM_EVENT_LOG_ADDR  "tpm_event_log_sm_addr"
```

Comment:
Are these documented anywhere? If so please add a link or reference.

Reply:
The name for the properties is still provisional, we have not made any final decision yet about how to call them so it is not documented anywhere yet.

Reply:
I wouldn't bother with defines for these two (makes the code more difficult to read for no benefit, the name is used only twice in less than 10 lines).